packages/assets-controllers/src/TokenRatesController.test.ts

```diff
@@ -272,8 +272,7 @@ describe('TokenRatesController', () => {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     tokenStateChangeListener!({ tokens: [], detectedTokens: [] });
 
-    // FIXME: This is now being called twice
```

Comment:
Removing the setters fixed this bug! The `configure` call was triggering both the chain ID and the tokens setter, resulting in a double update